include/infinispan/hotrod/DotNetClientListener.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * ClientListener.h
+ *
+ *  Created on: Aug 3, 2016
+ *      Author: rigazilla
+ */
+
+#ifndef INCLUDE_INFINISPAN_HOTROD_CACHECLIENTLISTENER_H_
+#define INCLUDE_INFINISPAN_HOTROD_CACHECLIENTLISTENER_H_
+
+#include "infinispan/hotrod/ClientEvent.h"
+#include "infinispan/hotrod/ImportExport.h"
+#include "infinispan/hotrod/ClientListener.h"
+#include "infinispan/hotrod/RemoteCacheBase.h"
+#include "hotrod/impl/transport/Transport.h"
+#include <vector>
+#include <list>
+#include <functional>
+
+using namespace infinispan::hotrod;
+
+namespace infinispan {
+namespace hotrod {
+
+namespace protocol {
+class Codec20;
+}
+template <class K, class V>
+class RemoteCache;
+
+namespace event {
+
+template <class K, class V>
+class DotNetClientListener
+{
+public:
+	
+	DotNetClientListener(RemoteCache<K,V>& cache) {};
```

Comment:
The ifdefs mention this should be CacheClientListener, but here it's DotNetClientListener